ci/get-ocp-repo.sh

```diff
@@ -25,20 +29,24 @@ info() {
     echo "INFO:" "$@" >&2
 }
 
-if [ $# -eq 0 ]; then
-    print_usage_and_exit
-else
-    mode=$1; shift
-    cosa_workdir=
-    ocp_manifest=
-    if [ "$mode" = "--cosa-workdir" ]; then
-        cosa_workdir=$1; shift
-    elif [ "$mode" = "--ocp-layer" ]; then
-        ocp_manifest=$1; shift
-    else
-        print_usage_and_exit
-    fi
-fi
+cosa_workdir=
+ocp_manifest=
+output_dir=
+rc=0
+options=$(getopt --options h --longoptions help,cosa-workdir:,ocp-layer:,output-dir: -- "$@") || rc=$?
+[ $rc -eq 0 ] || print_usage_and_exit
+eval set -- "$options"
+while [ $# -ne 0 ]; do
+    case "$1" in
+        -h | --help) print_usage_and_exit;;
+        --cosa-workdir) cosa_workdir=$2; shift;;
+        --ocp-layer) ocp_manifest=$2; shift;;
+        --output-dir) output_dir=$2; shift;;
+        --) break;;
+        *) echo "$0: invalid argument: $1" >&2; exit 1;;
+    esac
+    shift
+done
```

Comment:
previously `--cosa-workdir` and `--ocp-layer` were exclusive, or at least only the first one would be considered. Do we want to do anything specific if a user passed both here?

Reply:
Yeah... when switching to getopt, keeping that check didn't feel worth the lines. The current behaviour of just letting `--ocp-layer` win seemed fine to me. This script doesn't need a lot of polish. All users of it are in this repo.